org.newerth.savagexr.json

```diff
@@ -0,0 +1,121 @@
+{
+	"app-id": "org.newerth.savagexr",
+	"branch": "master",
+	"runtime": "org.freedesktop.Platform", 
+	"sdk": "org.freedesktop.Sdk", 
+	"runtime-version": "1.6", 
+	"tags": ["proprietary"],
+	"command": "bootstrapsavagexr.sh",
+	"build-options": {
+		"cflags": "-O3 -pipe",
```

Comment:
Why `-O3`?

Reply:
Removed, meant for making a faster version of SDL, but as you recommend shared modules I've removed it.